gems/activerecord/6.0/activerecord.rbs

```diff
@@ -724,10 +724,700 @@ end
 
 module ActiveRecord
   module Associations
-    class CollectionProxy
-      def build: (?::Hash[untyped, untyped] attributes) -> untyped | ...
-      def create: (?::Hash[untyped, untyped] attributes) -> untyped | ...
-      def create!: (?::Hash[untyped, untyped] attributes) -> untyped | ...
+    # Collection proxies in Active Record are middlemen between an
```

Comment:
At first, I moved this definition from activerecord-generated.rbs to here, and merge the above definition into it.
https://github.com/ruby/gem_rbs_collection/pull/598/commits/c68dd60bee8e9691b616937d9d8be80f89cd679c